### ISSUE TITLE
ArgumentOutOfRangeException with OdinsQOL's quick slots

### ISSUE THREAD

**Issue by FallenUpwards** [thedefside/BetterUI#28]
Hello,

When putting an item into any of the quick slots provided by OdinsQOL i get the following exception spam logged:

```
ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
  at System.ThrowHelper.ThrowArgumentOutOfRangeException (System.ExceptionArgument argument, System.ExceptionResource resource) [0x00029] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at System.ThrowHelper.ThrowArgumentOutOfRangeException () [0x00000] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at BetterUI.GameClasses.BetterHotkeyBar.PatchHotkeyBar (HotkeyBar& __instance, Player& player) [0x0003a] in <ba9fa52f025a4f1aab3fa5a0598003a4>:0 
  at (wrapper dynamic-method) HotkeyBar.DMD<HotkeyBar::UpdateIcons>(HotkeyBar,Player)
  at HotkeyBar.Update () [0x000d3] in <035307060cbb4b30b916cd82ebd80490>:0 
```

I have not yet tested if this happens with Randyknapp's EAQS too since he's still working on updating that one anyways.
Edit: Confirmed not happening with EAQS 2.1 (pulled directly from his repo).

Love the mod and been using it for practically ages, thanks for the quick update for Mistlands!

Kind regards
Fallen

**Comment by thedefside** [comment]
@FallenUpwards would you be willing to try the latest pre-release version with OdinQOL? The issue should be fixed now.

https://github.com/thedefside/BetterUI/releases/tag/v2.3.3-pre-release

**Comment by FallenUpwards** [comment]
> @FallenUpwards would you be willing to try the latest pre-release version with OdinQOL? The issue should be fixed now.
> 
> https://github.com/thedefside/BetterUI/releases/tag/v2.3.3-pre-release

Yep, can confirm no errors with Odin's quickslots anymore - at least during the small test i did just now.
I'm usually using EAQS anyways and was only trying out Odins' module while Randy was still working on the Mistlands compatibility update but i thought i'd report it anyways.

**Comment by thedefside** [comment]
resolved by #41
